prisma/schema.prisma

```diff
@@ -9,6 +9,7 @@
 
 generator client {
   provider = "prisma-client-js"
+  binaryTargets = ["debian-openssl-3.0.x", "debian-openssl-1.1.x"]
```

Comment:
We use Ubuntu 22.04 (jammy). So perhaps OpenSSL 3.0 is enough for us?

Reply:
basically yes, but there are some strange compability problems... I'll try solving this.